pennylane/tape/tape.py

```diff
@@ -1272,6 +1272,7 @@ def copy(self, copy_operations=False, tape_cls=None):
 
         tape._update()
         tape.trainable_params = self.trainable_params.copy()
+        tape._output_dim = self.output_dim
```

Comment:
This was a bug that was uncovered.

Reply:
nice! What was the bug? I haven't gotten to the tests yet, but have you added an extra test case for it?

Reply:
Oh I see that you have modified an existing test to take it into account :)